tracer/test/Datadog.Trace.Security.Unit.Tests/IAST/HashTests.cs

```diff
@@ -0,0 +1,65 @@
+// <copyright file="HashTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.Iast;
+using FluentAssertions;
+using Xunit;
+
+namespace Datadog.Trace.Security.Unit.Tests.Iast;
+
+public class HashTests
```

Comment:
It's not clear to me how these tests relate to the change being implemented, could you add a note in the description to explain it?

Reply:
Flavien was having strange behaviors whith the hashes in macos. This changes tryed to help him find out what was happening. I added the tests here because hash values changed in this PR